src/modules/events/events.service.ts

```diff
@@ -8,9 +8,33 @@ import { CreateEventDto } from './dto/create-event.dto';
 export class EventsService {
   constructor(private readonly prismaService: PrismaService) {}
 
-  create(createEventDto: CreateEventDto): Promise<Event> {
+  create(createEventDto: CreateEventDto, user: User): Promise<Event> {
+    const { id: userId } = user;
+    const eventData = {
+      ...createEventDto,
+      startTime: new Date(createEventDto.startTime),
+      endTime: new Date(createEventDto.endTime),
+    };
+
     return this.prismaService.event.create({
-      data: createEventDto,
+      data: {
+        ...eventData,
+        createdBy: userId,
```

Comment:
- ném `createdBy` vào trong `eventData` luôn em nhé
```suggestion
        ...eventData,
```